app/src/main/java/com/github/muellerma/mute_reminder/MediaAudioManager.kt

```diff
@@ -5,16 +5,27 @@ import android.media.AudioDeviceInfo
 import android.media.AudioManager
 import android.os.Build
 import android.util.Log
+import android.widget.Toast
 import androidx.core.content.getSystemService
 
-class MediaAudioManager(context: Context) {
+class MediaAudioManager(private val context: Context) {
     private val audioManager: AudioManager = context.getSystemService()!!
     private val prefs = Prefs(context)
 
     fun shouldNotify(): Boolean {
         return (!prefs.notifyOnlyWhenMuted || isRingToneMuted()) && !isMediaMuted() && !usesRemoteOutput()
     }
-
+    fun muteMedia():Boolean{
+        try {
+            audioManager.setStreamVolume(AudioManager.STREAM_MUSIC,0,AudioManager.FLAG_SHOW_UI)
```

Comment:
```suggestion
            audioManager.setStreamVolume(AudioManager.STREAM_MUSIC, 0, AudioManager.FLAG_SHOW_UI)
```

and below when creating the toast.